### ISSUE TITLE
Launch error: SlimJar failed to inject dependency

### ISSUE THREAD

**Issue by Jakub-Markowiak** [Lorenzo0111/ElectionsPlus#17]
Hello, everytime I try to launch my 1.17.1 Spigot server i get this error:

```
[18:12:42] [Server thread/INFO]: [ElectionsPlus] Enabling ElectionsPlus v1.3
[18:12:42] [Server thread/INFO]: [ElectionsPlus] Loading libraries..
[18:12:42] [Server thread/INFO]: [ElectionsPlus] Note: This might take a few minutes on first run.
[18:12:45] [Server thread/ERROR]: Error occurred while enabling ElectionsPlus v1.3 (Is it up to date?)
io.github.slimjar.injector.InjectionFailedException: SlimJar failed to inject dependency: name -> Dependency{groupId='mysql', artifactId='mysql-connector-java', version='8.0.25', snapshotId='null', transitive=[Dependency{groupId='com.google.protobuf', artifactId='protobuf-java', version='3.11.4', snapshotId='null', transitive=[]}]}
	at io.github.slimjar.injector.SimpleDependencyInjector.injectDependencies(SimpleDependencyInjector.java:63) ~[?:?]
	at io.github.slimjar.injector.SimpleDependencyInjector.inject(SimpleDependencyInjector.java:50) ~[?:?]
	at io.github.slimjar.app.builder.InjectingApplicationBuilder.build(InjectingApplicationBuilder.java:55) ~[?:?]
	at me.lorenzo0111.pluginslib.dependency.slimjar.SlimJarDependencyManager.build(SlimJarDependencyManager.java:28) ~[?:?]
	at me.lorenzo0111.elections.ElectionsPlus.load(ElectionsPlus.java:153) ~[?:?]
	at me.lorenzo0111.elections.ElectionsPlus.onEnable(ElectionsPlus.java:77) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:505) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:419) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:604) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:266) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:995) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[slavicraft17.jar:3203-Spigot-18c71bf-aa69d1c]
	at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.nio.file.NoSuchFileException: F:\Serwery\Minecraft%20-%20serwery\Slavicraft%20III%20Edycja%20Reanimacja%20Trupa\plugins\ElectionsPlus\libs\mysql\mysql-connector-java\8.0.25\relocated\ElectionsPlus\mysql-connector-java-8.0.25.jar
	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85) ~[?:?]
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103) ~[?:?]
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108) ~[?:?]
	at sun.nio.fs.WindowsFileAttributeViews$Basic.readAttributes(WindowsFileAttributeViews.java:53) ~[?:?]
	at sun.nio.fs.WindowsFileAttributeViews$Basic.readAttributes(WindowsFileAttributeViews.java:38) ~[?:?]
	at sun.nio.fs.WindowsFileSystemProvider.readAttributes(WindowsFileSystemProvider.java:199) ~[?:?]
	at java.nio.file.Files.readAttributes(Files.java:1843) ~[?:?]
	at java.util.zip.ZipFile$Source.get(ZipFile.java:1230) ~[?:?]
	at java.util.zip.ZipFile$CleanableResource.<init>(ZipFile.java:707) ~[?:?]
	at java.util.zip.ZipFile.<init>(ZipFile.java:241) ~[?:?]
	at java.util.zip.ZipFile.<init>(ZipFile.java:171) ~[?:?]
	at java.util.jar.JarFile.<init>(JarFile.java:348) ~[?:?]
	at java.util.jar.JarFile.<init>(JarFile.java:319) ~[?:?]
	at java.util.jar.JarFile.<init>(JarFile.java:258) ~[?:?]
	at io.github.slimjar.injector.loader.InstrumentationInjectable.inject(InstrumentationInjectable.java:70) ~[?:?]
	at io.github.slimjar.injector.SimpleDependencyInjector.injectDependencies(SimpleDependencyInjector.java:60) ~[?:?]
	... 15 more
```

**Comment by Lorenzo0111** [comment]
Hello, I already know this bug, I already opened an issue on the dependency injector repository: SlimJar/SlimJar#48

**Comment by Lorenzo0111** [comment]
Fixed in [v1.4](https://www.spigotmc.org/resources/electionsplus-add-democracy-to-your-server.93463/download?version=411856)
